packages/react/src/Campaign/Campaign.tsx

```diff
@@ -45,15 +45,18 @@ const CampaignInner = <T extends React.ElementType = 'div'>(
     <Component
       className={cx(
         className,
-        'grid gap-8 md:grid-flow-col md:grid-cols-[50%,50%] md:gap-0',
+        'container grid gap-8 sm:grid-flow-col sm:gap-0 sm:p-0 lg:grid-cols-[50%,50%] lg:p-4',
```

Comment:
Vil ikke dette låse campaign i å alltids være maks container-bredde? Vil ikke være fluid if et team har lyst til å ha bredere campaigns. Usikker på hva som er best selvsagt, men tenkte å høre uansett

cc: @alexanbj

Reply:
Hmm godt poeng. Tenker også det er nice å unngå for mye setting av størrelse og spacing i gjenbrukbare komponenter, men litt gråsone når spacingen rundt elementet skal forsvinne på mindre skjermer. Fulgte det som var gjort i hero.

Reply:
Okidoki, jeg godkjenner denne siden Bjerkan ikke har svart. Så får vi endre på det i etterkant hvis det merges inn før svar